winmake.ps1

```diff
@@ -0,0 +1,86 @@
+# Targets
```

Comment:
I'd recommend setting an error action to stop, so that any cmdlet/func errors will abort the script. Alternatively you could use try/catch.

```suggestion
$ErrorActionPreference = 'Stop'

# Targets
```